fiotest/api.py

```diff
@@ -86,19 +89,37 @@ def complete_test(
 
     @staticmethod
     def target_name(sota_dir: str) -> str:
-        with open(os.path.join(sota_dir, "current-target")) as f:
-            for line in f:
-                if line.startswith("TARGET_NAME"):
-                    k, v = line.split("=")
-                    return v.replace('"', "").strip()  # remove spaces and quotes
-        sys.exit("Unable to find current target")
+        try:
+            with open(os.path.join(sota_dir, "current-target")) as f:
+                for line in f:
+                    if line.startswith("TARGET_NAME"):
+                        k, v = line.split("=")
+                        return v.replace('"', "").strip()  # remove spaces and quotes
+        except FileNotFoundError:
+            sys.exit("Unable to find current target")
 
     @staticmethod
     def test_url(sota_dir: str) -> str:
-        with open(os.path.join(sota_dir, "sota.toml")) as f:
-            for line in f:
-                if line.startswith("server ="):
+        try:
+            with open(os.path.join(sota_dir, "sota.toml")) as f:
+                for line in f:
+                    if line.startswith("server ="):
+                        k, v = line.split("=")
+                        v = v.replace('"', "").strip()  # remove spaces and quotes
+                        return v + "/tests"
+        except FileNotFoundError:
+            sys.exit("Unable to find server url")
```

Comment:
you still need a `sys.exit("Unable to find server url")` at the end of this function

Reply:
Fixed